esx_service/utils/auth_data_const.py

```diff
@@ -0,0 +1,45 @@
+## VM based authorization for docker volumes
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+""" Define string constant for column name for table in authorization DB"""
+
+# column name in tenants table
+ID = 'id'
```

Comment:
Same comment - need to add prefixes. E.g. COL_ID COL_NAME COL_DESCRIPTION, etc. for columns